### ISSUE TITLE
Inspecting not available for classes deriving from abstract : BaseBehavior type classes

### ISSUE THREAD

**Issue by WiktorWasowski** [jacobdufault/fullinspector#173]
If I have a class like this one:
``` C#
using UnityEngine;
using System.Collections.Generic
using FullInspector;

public abstract class SomeClass : BaseBehavior
{
    public int field;
    public int property { get; set; }
    public Dictionary<int, int> dictionary;
}
```
And then a class that derives from it:
``` C#
using UnityEngine;
using System.Collections.Generic;

public class DerivedClass : SomeClass
{
    public int otherField;
    public int otherProperty { get; set; }
    public Dictionary<int, int> otherDictionary;
}
```
If I now plug `DerivedClass` to a GameObject, I can only inspect the fields `field` and `otherField` in the editor.

**Comment by jacobdufault** [comment]
This works as expected for me using the current master. Please reopen if you can reproduce using [this] (https://github.com/jacobdufault/fullinspector/releases/download/2.6.4/FullInspector-DLLs.zip) (or a more recent) build. Thanks!

![image](https://cloud.githubusercontent.com/assets/356714/19218987/9943f2b2-8dbc-11e6-9894-bb047f7685d6.png)

**Comment by WiktorWasowski** [comment]
I'm sorry. I wrote this code here to illustrate the problem in my own scripts in a short way, without testing it. It works indeed (with the same version I used before).
I found out what's the true source of the problem with my derived classes. They have editor scripts associated with them and I forgot I had to modify them in a Full Inspector way. Now that I've done that, everything works great.
